lib/binaries/chrome_xml.ts

```diff
@@ -175,11 +175,12 @@ export function getValidSemver(version: string): string {
     // no-op: is this is not valid, do not throw here.
   }
   // This supports downloading 74.0.3729.6
+  // Removing the 0 after 74, to keep patch version of webdriver
   try {
-    const newRegex = /(\d+.\d+.\d+).\d+/g;
+    const newRegex = /(\d+.)\d+.(\d+.\d+)/g;
     const exec = newRegex.exec(version);
     if (exec) {
-      lookUpVersion = exec[1];
+      lookUpVersion = exec[1] + exec[2];
```

Comment:
This is a good start but probably won't do it. I can see where we might have 74.0.5.5 and then we'll get 74.1.4.4. We will just grab 74.0.5.5 with your change. Possibly we could make two semver objs from chromedriver <a.b.c.d> we could have one that is a.b.c and another one that is b.c.d. We could check what is the a.b.c versions and then compare the b.c.d versions for the latest.

Reply:
Thanks, done